set.mm

```diff
@@ -51077,18 +51077,91 @@ That theorem bundles the theorems ( ` |- E. x ( x = y -> z e. x ) ` with
       ERUGBPEABUBBSTUFRUGAPEABUCASTUD $.
   $}
 
+  ${
+    $d x y z $.
+    $( There exist two sets, one a member of the other.
+
+       This theorem looks similar to ~ el , but its meaning is different.  It
+       only depends on the axioms ~ ax-mp to ~ ax-4 , ~ ax-6 , and ~ ax-pr .
+       This theorem does not exclude that these two sets could actually be one
+       single set containing itself.  That two different sets exist is proved
+       by ~ exex .  (Contributed by SN, 23-Dec-2024.) $)
+    exel $p |- E. y E. x x e. y $=
+      ( vz weq wo wel wi wal wex ax-pr ax6ev pm2.07 eximii exim mpi ) ACDZPEZAB
+      FZGAHZRAIZBCCBAJSQAITPQAACKPLMQRANOM $.
+    $( $j usage 'exel' avoids 'ax-5' 'ax-7' 'ax-8' 'ax-9' 'ax-10 'ax-11'
+       'ax-12' 'ax-13''ax-ext' 'ax-sep' 'ax-nul' 'ax-pow'; $)
+  $}
+
+  ${
+    $d x y z $.
+    $( There exist two different sets.  (Contributed by NM, 7-Nov-2006.)  Avoid
+       ~ ax-13 .  (Revised by BJ, 31-May-2019.)  (Revised by Gino Giotto,
+       5-Sep-2023.)  Avoid ~ ax-8 .  (Revised by SN, 21-Sep-2023.)  Avoid
+       ~ ax-12 .  (Revised by Rohan Ridenour, 9-Oct-2024.)  Use ~ ax-pr instead
+       of ~ ax-pow .  (Revised by BTernaryTau, 3-Dec-2024.)  Extract this
+       result from the proof from ~ dtru .  (Revised by BJ, 2-Jan-2025.) $)
+    exex $p |- E. x E. y -. x = y $=
```

Comment:
I'd prefer `exexneq` here instead of `exex`. I would like `exex` to be [bj-exexbiex](https://us.metamath.org/mpeuni/bj-exexbiex.html) when it will be moved in main. Notice that I called [exexw](https://us.metamath.org/mpeuni/exexw.html) with this idea in mind.